src/Nox.Types/Types/Image/TypeOptions/ImageTypeOptions.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Nox.Types;
+
+/// <summary>
+/// Represents the options for image types.
+/// </summary>
+public class ImageTypeOptions
+{
+    /// <summary>
+    /// Default format type.
+    /// </summary>
+    private static ImageFormatType DefaultFormatType => ImageFormatType.All;
+
+    /// <summary>
+    /// Gets or sets the format types of the image.
+    /// </summary>
+    public List<ImageFormatType> ImageFormatTypes { get; set; } = new() { DefaultFormatType };
```

Comment:
Not sure how our schema will understand the ImageFormatType, we need to evaluate this

Create the property  ImageTypeOptions in NoxSimpleTypeDefinition

Generate the schema and validate if the result is the expected

Reply:
ImageFormatType enum was added and decorated with custom ImageFormat enum.